### DISCUSSION TITLE
[Feature Request] Can add the Message Component

### DISCUSSION THREAD

**Discussion by Tuan2002** [heroui-inc/heroui#1312] [Feature requests]
### Describe the solution you'd like

Now, I still use custom modal component to display message or notification. 
So, I would like it is an native component to ez using and improve the performance

### Describe alternatives you've considered

That component can has some props e.g (type, duration, children, config).
Thanks!

### Screenshots or Videos

<img width="595" alt="Screen Shot 2023-08-08 at 09 30 14" src="https://github.com/nextui-org/nextui/assets/64023863/71415294-bf35-482c-9760-2192241b6295">

**Comment by ibalbal** [comment]
The project needs to contain a message component to alert the user to the status of the action
